release_notes.md

```diff
@@ -0,0 +1,67 @@
+ # Release notes
+
+- Woof Mobile APP
+- Release notes version: 1
+- Release note date: May. 4, 2023
+
+All notable changes to this application will be documented in this file.
+
+## Overview
+
+This is the application for iOS.
```

Comment:
Why do we have app overview in the release notes?

Reply:
My bad. The unnecessary information are removed.

Reply:
I found some article with release notes' examples and jut try to make the same file. 
For example:
https://www.indeed.com/career-advice/career-development/release-notes-template
But , if in your opinion, this information is not needed it's ok for me. The paragraph "Overview" is deleted.